xmppserver/src/main/java/org/jivesoftware/openfire/server/ServerDialback.java

```diff
@@ -395,7 +398,7 @@ public LocalIncomingServerSession createIncomingSession(XMPPPacketReader reader)
                 if ("db".equals(doc.getNamespacePrefix()) && "result".equals(doc.getName())) {
                     String hostname = doc.attributeValue("from");
                     String recipient = doc.attributeValue("to");
-                    Log.debug("ServerDialback: RS - Validating remote domain for incoming session from {} to {}", hostname, recipient);
+                    Log.debug("ServeDialback: RS - Validating remote domain for incoming session from {} to {}", hostname, recipient);
```

Comment:
This should remain "ServerDialback" (unsure why you removed the 'r'?)

Reply:
This hasn't been fixed yet.